src/Backend/Core/Js/Components/Forms.js

```diff
@@ -409,8 +288,43 @@ export class Forms {
 
   // Add date pickers to the appropriate input elements
   datePicker () {
+    const dayNames = [
+      window.backend.locale.loc('DayLongSun'), window.backend.locale.loc('DayLongMon'), window.backend.locale.loc('DayLongTue'),
```

Comment:
shouldn't we just change this to moment.js or something like that?

Reply:
I wanted to keep the scope of this PR small and focus just on the date/time picker. The current implementation works and doesn't add another dependency, so I figured it was fine as is.